canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -368,13 +373,13 @@ class ToolTip extends React.Component {
 
 		if ((typeof this.props.tip) === "string") {
 			tooltipContent = (
-				<span id="tooltipContainer">
+				<span id={`${this.uuid}-tooltipContainer`}>
```

Comment:
For the `id` do we actually need to set `id`?  I'm curious if it's actually needed.  This seems like it should be a class or not set.